### ISSUE TITLE
Design suggestions

### ISSUE THREAD

**Issue by Jesterjke** [kirigayakazushin/deepin-topbar#68]
Just updated from 0.4
A few personal thoughts about update:
1. The way that you decided to fill topbar with fully black color on fullscreen apps looks wrong for me. It much looked better previously, when it was semi-transparent all the time. You could probably make it a little darker on full screen, I think.
2. Speed indicator - now is much better.
3. Sound and battery indicator - worse. The previous ones looked like more material stylish and the current are like from windows98 (dont take it personally, just my opinion)

**Comment by aiamuzz** [comment]
Yes ... the topbar turns black when a full screen app is running ... i agree a transparent one is preferred
full marks to the speed indicator ... it looks fantastic !!!

Is it possible for you to use the same sound and battery icons from the deepin efficient mode bar ... the same icons that show in the efficient bar system tray ?
